### ISSUE TITLE
Terraform appears to be out of date

### ISSUE THREAD

**Issue by mcfearsome** [hashicorp/terraform-k8s#64]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### terraform-k8s & Kubernetes Version

terraform-k8s: 0.1.5-alpha
kubernetes: 1.15

### Affected Resource(s)

Operator

### Debug Output

```
"could not read state file, Error, state snapshot was created by Terraform v0.13.2, which is newer than current v0.12.25; upgrade to Terraform v0.13.2 or greater to work with this state"
```

### Expected Behavior

The operator to be kept up to date with its state store.

### Actual Behavior

Most times a new workspace will be created but not actually run and I am queuing them manually. It does run sometimes but it is not consistent. Attempting to update a workspace didn't appear to work at all but I did not actually check the variables in the TFC UI before deleting and starting over.

### Steps to Reproduce

try to create a workspace

### Important Factoids

Is this project alive? I wish I had the time to spend on it, it might be more encouraging if there was active development happening.

**Comment by Redeux** [comment]
Hi @mcfearsome.  Thanks for the issue, as always.  We are accelerating our development of the operator and we'll take a look at this issue.

**Comment by koikonom** [comment]
Fixed in #77
